events.ts

```diff
@@ -128,61 +157,159 @@ const cover = {
   }
 
   function run() {
-    try {
+    cover.waitFor(
+      '.Notice.Notice--info.Notice--animated.Notice--center',
```

Comment:
make an enum for all the labels. Don't put hardcoded text inside functions

Reply:
I see your point. But this code is alla bit temporary and will change often, so I'm thinking about other solutions like #36, so for now I move this to an issue instead of part of this pull request. New issue is #37.